components/TripDetailView.js

```diff
@@ -1,17 +1,12 @@
 import { styled } from "styled-components";
 import PrimaryLink from "./PrimaryLink";
 
-export default function TravelOverview({
-  trip,
-  handleDelete,
-  handleEdit,
-  tripId,
-  todos,
-}) {
+export default function TravelOverview({ trip, handleDelete, tripId, todos }) {
```

Comment:
This is just about the naming: Your file has the name `TripDetailView` but the name of your component is `TravelOverview` and I would prefer to choose one name. :)